package.json

```diff
@@ -191,6 +191,7 @@
     "event-stream": "3.3.4",
     "express": "^4.13.3",
     "fast-boot2": "^1.0.9",
+    "get-stdin": "^7.0.0",
```

Comment:
The previous `capitano.utils.getStdin` was just an alias to get-stdin 4.0.1, so I switched to using it directly and also updated to include promise+typescript support